lib/main.dart

```diff
@@ -3,50 +3,61 @@ import 'dart:io';
 import 'package:destiny/views/shared/receive.dart';
 import 'package:destiny/views/shared/send.dart';
 import 'package:dart_wormhole_william/client/native_client.dart';
+import 'package:fimber/fimber.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
 import 'package:provider/provider.dart';
 import 'package:window_size/window_size.dart';
+import 'package:get_it/get_it.dart';
 
 import 'config/routes/routes_desktop_config.dart';
 import 'config/routes/routes_mobile_config.dart';
 import 'config/theme/custom_theme.dart';
 import 'constants/app_constants.dart';
+import 'locator.dart';
+
+// This is our global ServiceLocator
+GetIt getIt = GetIt.instance;
 
 void startApp(Config c) {
   WidgetsFlutterBinding.ensureInitialized();
+  Fimber.plantTree(DebugTree());
```

Comment:
Is this applicable only for --debug or also will be used in --release build?

Reply:
We can remove Fimber for now.